avocado_vt/test.py

```diff
@@ -406,7 +405,8 @@ def _runTest(self):
                         raise error.TestWarn("funcatexit failed with: %s" %
                                              error_message)
 
-                except Exception:
+                except:  # nopep8 Old-style exceptions are not inherited from Exception()
```

Comment:
OK, I see the goal here.  I'll spend some time here (after I post this initial review) to test it thoroughly.

Reply:
it goes from exceptions I observed while executing `io-github-autotest-qemu` tests (all)